Makefile.org

```diff
@@ -424,6 +424,14 @@ clean:	libclean
 	rm -fr $$i/*; \
 	done
 
+distclean: clean
+	-$(RM) `find . -type l -a \! -path "./.git/*"`
+	$(RM) apps/CA.pl
+	$(RM) test/evptests.txt test/newkey.pem test/testkey.pem test/testreq.pem
+	$(RM) tools/c_rehash
+	$(RM) crypto/opensslconf.h
+	$(RM) Makefile Makefile.bak
```

Comment:
>  And yes, it means that you have to add empty distclean: rules in a bunch of Makefiles. It also means that you won't have to deal with a possible .git directory ;-)

I see the point, but there are 72 Makefile's in the tree.  And I doubt it would be worth the pain,
because the distclean target is only a best effort kind of thing, so if it don't work out right,
it is at least guaranteed to not break anything.

That said, is there a chance that the PR will be accepted if I use the slightly more portable
find syntax above?